lib/std/zig/render.zig

```diff
@@ -1159,8 +1159,29 @@ fn renderContainerField(
         try renderToken(ais, tree, rparen_token, .space); // )
     }
     const eq_token = tree.firstToken(field.ast.value_expr) - 1;
-    try renderToken(ais, tree, eq_token, .space); // =
-    return renderExpressionComma(gpa, ais, tree, field.ast.value_expr, space); // value
+    const eq_space: Space = if (tree.tokensOnSameLine(eq_token, eq_token + 1)) .space else .newline;
+    {
+        ais.pushIndent();
+        try renderToken(ais, tree, eq_token, eq_space); // =
+        ais.popIndent();
+    }
+
+    if (eq_space == .space)
+        return renderExpressionComma(gpa, ais, tree, field.ast.value_expr, space); // value
+
+    const token_tags = tree.tokens.items(.tag);
+    const maybe_comma = tree.lastToken(field.ast.value_expr) + 1;
+
+    if (token_tags[maybe_comma] == .comma) {
+        ais.pushIndent();
+        try renderExpression(gpa, ais, tree, field.ast.value_expr, .none); // value
+        ais.popIndent();
+        try renderToken(ais, tree, maybe_comma, space);
+    } else {
+        ais.pushIndent();
+        try renderExpression(gpa, ais, tree, field.ast.value_expr, space); // value
+        ais.popIndent();
+    }
```

Comment:
This is written this way to handle indentation for line comments directly after the comma correctly I take it? It would probably be good to have a test case for this behavior.

Reply:
Not quite, `renderExpressionComma` renders the comma after the multiline string one indentation level too far.

Reply:
Ah that makes sense, though it will also fix this case:
```zig
const Foo = struct {
    bar: u32 =
        42,
    // comment
```
which will currently render as the following if I'm accurately running this code in my head:
```zig
const Foo = struct {
    bar: u32 =
        42,
        // comment
```

Reply:
Nope, the comment is rendered at the same level as `bar`. That's unexpected :thinking:

Reply:
Oh I think I wasn't very clear, I expect your current code to render the comment on the same level as bar. Without this special handling for the comma that you added in the latest commit though I'd expect the comment to be rendered at the wrong indentation level.

Reply:
All clear then, I'll shortly push another commit with the extra test